admin/partials/perfecty-push-admin-post-metabox.php

```diff
@@ -1,6 +1,18 @@
-<input type="checkbox"
+<p><input type="checkbox"
 	   id="perfecty_push_send_on_publish"
 	   name="perfecty_push_send_on_publish"
 	   <?php if ( $send_notification === true ) {
 		   echo 'checked="checked"';} ?>"/>
-<label for="perfecty_push_send_on_publish"><?php printf( esc_html__( 'Send notification on publish', 'perfecty-push-notifications' ) ); ?></label>
+<label for="perfecty_push_send_on_publish"><?php printf( esc_html__( 'Send notification on publish', 'perfecty-push-notifications' ) ); ?></label></p>
+<p><input type="checkbox"
+	   id="perfecty_push_send_featured_img"
+	   name="perfecty_push_send_featured_img"
+	   <?php if ( $send_featured_img === true ) {
+		   echo 'checked="checked"';} ?>"/>
+<label for="perfecty_push_send_featured_img"><?php printf( esc_html__( 'Send featured image', 'perfecty-push-notifications' ) ); ?></label></p>
```

Comment:
I like the idea of making it easier for the user to get the first post image in case they don't have a post thumbnail. Should we make it clear in the text we use here that if the thumbnail not present, we will fetch the first image from the content? Like `'Send featured image or first image found in content'`?

Reply:
Ok, but I would like to discuss about the image size because thumbnails can be really small (default is 150x150).
Do you think should we use another size or define a custom size?
Or, could we add an option to choose image size?

Reply:
For the moment we can use the default size, and if needed we can implement what you suggested. I think 150x150 is acceptable.

Reply:
Do you consider it's a good idea to just use the featured image and don't do the first image from the post? In case no featured image is found, the fallback will still be the blog icon/image, which is not a bad thing.

Reply:
I have seen that many themes use first image in the post as a post thumbnail on blog main page when no featured image is set. So I think that using it in notification is in some way an expected behaviour.
More, perfecty-push always shows $icon = get_site_icon_url() in every notification, so I think that sending an image from the post, when we can find it, is a good idea.
What I am thinking about is what image size to use when:
- the theme has no "post-thumbnail" size declared because it doesn't support post-thumbnails, and
- the user asks to send "featured image" in notifications.
In this case, there should never be a featured image set in post (because the theme doesn't allow for it and featured image interface should not appear on the edit screen) but we could have a first image to send.
At the end, I am thinking that in this scenario the best thing could be to send original size image in notifications (I have seen that many themes declare post-thumbnail support, but don't create a new specific image size to fit it; so, even when featured image is set, we are sending an original size image in most cases), but I am still making some tests with different themes and "strange" images, to see what can happen.